storage-resize-images/extension.yaml

```diff
@@ -156,3 +156,21 @@ params:
       If you prefer not to overwrite or add `Cache-Control` metadata, leave this field empty.
     example: max-age=86400
     required: false
+
+  - param: IMAGE_TYPE
+    label: Image type to convert to
```

Comment:
This might need rewording to something like: `Convert image to preferred type`. Thoughts @laurenzlong ?